files/build_templates/chassis-module.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Chassis module up & down operation
+ConditionPathExists=/etc/sonic/chassisdb.conf
+Requires=database.service updategraph.service
```

Comment:
updategraph.service is a 202205 service which does not exist in 202405 (master). This is for 202205 cherry-pick purpose. We need another PR to change this in master after this PR fix has been picked up by 202205. Any better strategy here?

Reply:
@mlok-nokia , can you comment on this?

Reply:
No longe applicable